src/other/Table.css

```diff
@@ -6,13 +6,18 @@
 }
 
 .other_Table th {
-    padding:5px 10px;
+	padding:5px 10px;
 	background-color:#1f77b4;
 	border-width: 1px;
 	border-style: solid;
 	border-color: #a9c6c9;
```

Comment:
My text editor (sublime text), doesn't seem to show anything wrong, nor does vim :/
![selection_005](https://cloud.githubusercontent.com/assets/6505326/7818913/1c6f67d0-03fc-11e5-8f87-4694bc9633ba.png)
![selection_006](https://cloud.githubusercontent.com/assets/6505326/7818915/200ab142-03fc-11e5-9284-82aca60edc83.png)